1. 'IGROV1_example_no_growth_thresholds.ipnyb' is an example of a model built with no growth thresholds, constrained for media conditions and gene expression data (CCLE; 2020). Gene expression and media conditions are cell line-specific, according to 'media_subtype_site_growth_all_datasets.xls' supplementary file. Corresponds to Results 3.2.
2. 'IGROV1_example_with_growth_thresholds.ipnyb' is an example of a model built with growth thresholds, according to cell line-specific experimental data. Constrained for media conditions, gene expression data (CCLE; 2020) and these groth thresholds. Details and references for media conditions and experimental growths included in 'media_subtype_site_growth_all_datasets.xls' supplementary file. Corresponds to Results 3.2.
3. 'IGROV1_multiomics_with_proteomics_non-scaled.ipnyb' is an example of a model built with growth thresholds, and constrained with both gene expression (CCLE; 2020) and protein expression data (CCLE; 2020). Also contrained with media conditions. In this example, proteomics has not been scaled to within range of gene expression data, but has been non-negative transformed. Corresponds to Results 3.3. 
4. 'IGROV1_multiomics_with_proteomics_scaled.ipnyb' is an example of a model built with growth thresholds, and constrained with both gene expression (CCLE; 2020) and protein expression data (CCLE; 2020). Also contrained with media conditions. In this example, proteomics has been scaled to within range of gene expression data, (as well as non-negative transformed). Corresponds to Results 3.3. 
5. 'Building_n=39_GIMME_models.ipynb' shows how GIMME was implemented (within MEWpy) to constrain n=39 ovarian metabolic models, for comparison against the output from my single-omics-integration algorithm (notebook file 'IGROV1_example_with_growth_thresholds.ipnyb'). Models constrained using gene expression data (CCLE; 2018) and media conditions. Corresponds to Results 3.4.
