### Link to publication of PhD work: https://doi.org/10.1038/s41540-024-00418-5
![image](https://github.com/user-attachments/assets/a2dc1862-6591-46ff-b0f6-6b3319e59c2c)
![image](https://github.com/user-attachments/assets/8c4366a8-e8ef-4604-96c5-f7f10699687f)


### Work presented here has been published in Systems Biology and Applications, August 2024 (Meeson and Schwartz): https://lnkd.in/epkMwiTP 'Constraint-based modelling predicts metabolic signatures of low and high-grade serous ovarian cancer'

### Work has been included in the pre-print: https://www.biorxiv.org/content/10.1101/2025.06.24.661329v1

In 'algorithm_development' folder
  1. 'IGROV1_example_no_growth_thresholds.ipnyb' is an example of a model built with no growth thresholds, constrained for media conditions and gene expression data (CCLE; 2020). Gene expression and media conditions are cell line-specific, according to 'media_subtype_site_growth_all_datasets.xls' supplementary file. Corresponds to Results 3.2 of Thesis.
  2. 'IGROV1_example_with_growth_thresholds.ipnyb' is an example of a model built with growth thresholds, according to cell line-specific experimental data. Constrained for media conditions, gene expression data (CCLE; 2020) and these groth thresholds. Details and references for media conditions and experimental growths included in 'media_subtype_site_growth_all_datasets.xls' supplementary file. Corresponds to Results 3.2 of Thesis.
  3. 'IGROV1_multiomics_with_proteomics_non-scaled.ipnyb' is an example of a model built with growth thresholds, and constrained with both gene expression (CCLE; 2020) and protein expression data (CCLE; 2020). Also contrained with media conditions. In this example, proteomics has not been scaled to within range of gene expression data, but has been non-negative transformed. Corresponds to Results 3.3 of Thesis. 
  4. 'IGROV1_multiomics_with_proteomics_scaled.ipnyb' is an example of a model built with growth thresholds, and constrained with both gene expression (CCLE; 2020) and protein expression data (CCLE; 2020). Also contrained with media conditions. In this example, proteomics has been scaled to within range of gene expression data, (as well as non-negative transformed). Corresponds to Results 3.3 of Thesis. 
  5. 'Building_n=39_GIMME_models.ipynb' shows how GIMME was implemented (within MEWpy) to constrain n=39 ovarian metabolic models, for comparison against the output from my single-omics-integration algorithm (notebook file 'IGROV1_example_with_growth_thresholds.ipnyb'). Models constrained using gene expression data (CCLE; 2018) and media conditions. Corresponds to Results 3.4 of Thesis.
  6. 'Building_OCM-specific_models.ipnyb' shows how OCM-specific models were constrained, according to a dataframe containing experimental growth thresholds based on experimental growth rates and gene expression values (data generated by Taylor lab University of Manchester; Nelson et al, 2020). Notebook includes single-omics-integration algorithm (notebook file 'IGROV1_example_with_growth_thresholds.ipnyb'), which has been wrapped as a function to make it easy to generate a large number of models. Notebook also contains information on how OCM gene expression data has been scaled to within range of CCLE 2020 gene expression data, so that a similar range of constraints are being applied to Human1.

In 'media_conditions' folder are CSV files for all media conditions specified throughout this project.

In 'cell_line_annotations' folder are CCLE cell lines, with subtype, media and growth rate, including references.

In 'ocm_patient_modelling' folder is the code to iterate through large RNAseq dataset and constrain patient-specific models, using final single-omics integration algorithm, as well as model fluxes and cluster assignments following k-means clustering.

In 'siTPI1_validation' folder is a dataframe of model fluxes from Human1 constrained using NTsiRNA-specific or siTPI1-specific RNAseq data from experimental siRNA transfection, and a model built using NTsiRNA expression data with TPI1 knockout simulated within model.
